### PR TITLE
Minor adjustment on sliding menu & settings page

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/SlidingMenu.vue [hunk 1]
```diff
@@ -104,8 +104,20 @@ export default {
...
       'getUserCivicLikerHalo',
+      'getUserIsCivicLiker',
+      'getUserIsCivicLikerTrial',
       'getHomeRoute',
     ]),
+
+    menuItemTitleForCivic() {
+      if (
+        (this.getUserId && !this.getUserIsCivicLiker) ||
+        this.getUserIsCivicLikerTrial
```
Comment (williamchong007) [inline src/components/SlidingMenu.vue]: `this.getUserId && (!this.getUserIsCivicLiker || this.getUserIsCivicLikerTrial)` ?

Comment (nwingt) [reply src/components/SlidingMenu.vue]: The condition is Liker OR Civic Liker (Trial)

Comment (williamchong007) [reply src/components/SlidingMenu.vue]: but one cannot be a `Civic Liker (Trial)` without `getUserId()` being true?

Comment (nwingt) [reply src/components/SlidingMenu.vue]: How about I wait for #126 to merge and I will use the new getter...

Comment (williamchong007) [reply src/components/SlidingMenu.vue]: port the getter from #126 instead? this PR seems simpler and should merge first

Comment (nwingt) [reply src/components/SlidingMenu.vue]: Ported

**Pull request merged**